docs/source/dashboard.rst

```diff
@@ -61,27 +80,53 @@ indicate the following.
+            <td>
+                <div role="img" aria-label="orange square" style="color:rgba(255, 165, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td> Memory is close to the spilling to disk target by at least one worker - (default above 50% of memory available)</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="red square" style="color:rgba(255, 0, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>When it pauses or enters its retirement stage - above 100% target threshold</td>
+            <td>
+                <div role="img" aria-label="red square" style="color:rgba(255, 0, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>When at least one worker paused or in retiring stage - above 100% of target threshold</td>
         </tr>
         <tr>
             <td>
-                <div role="img" aria-label=" grey square" style="color:rgba(128, 128, 128, 1); font-size: 25px ">&#9632;</div>
+                <div role="img" aria-label="grey square" style="color:rgba(128, 128, 128, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Memory spilled to disk</td>
+            <td>
+                <div role="img" aria-label="grey square" style="color:rgba(128, 128, 128, 1); font-size: 25px ">&#9632;</div>
             </td>
             <td>Memory spilled to disk</td>
         </tr>
     </table>
 
-.. figure:: images/dashboard_memory.png
-    :alt: Two bar charts on memory usage. The top chart shows the total cluster memory in a single bar with mostly under target memory in blue and a small part of spilled to disk in grey. The bottom chart displays the memory usage per worker, with a separate bar for each of the 16 workers. The first four bars are orange as their worker's memory are close to the spilling to disk target, with the first worker standing out with a portion in grey that correspond to the amount spilled to disk. The remaining workers are all under target showing blue bars.
+.. figure:: images/dashboard_memory_new.gif
+    :alt: Two bar charts on memory usage. The top chart shows the total cluster memory in a single bar with mostly under target memory - changing to colours according to memory usage, (blue - under target, orange - Memory is about to be spilled (above 50%), red - paused or in retirement stage (above 100%), and a small part of spilled to disk in grey. The bottom chart displays the memory usage per worker, with a separate bar for each of the 4 workers. The four bars are can be seen in various colours as in blue when under target, orange as their worker's memory are close to the spilling to disk target, with the second and fourth worker standing out with a portion in grey that correspond to the amount spilled to disk, also fourth worker in red is paused or about to retire. also all worker be seen as orange at a time when all are above the 50% of target memory spiiled amount.
```

Comment:
- typo in last sences "spiiled" -> "spilled"
- I think the percentages here are a bit off. Asking the person that worked on this for advice.

Reply:
I have updated the correct spelling for spilled. and updated the description.